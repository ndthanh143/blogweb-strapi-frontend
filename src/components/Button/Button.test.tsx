import React from 'react';
import renderer, { ReactTestRendererJSON } from 'react-test-renderer';
import { Button } from './index';

describe('Button component', () => {
  test('should render correctly', () => {
    const component = renderer.create(<Button variant="solid">Hello</Button>);
    let tree = component.toJSON() as ReactTestRendererJSON;
    expect(tree).toMatchSnapshot();
  });
});
