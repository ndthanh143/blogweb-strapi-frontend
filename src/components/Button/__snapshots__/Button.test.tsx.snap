// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component should render correctly 1`] = `
<button
  className="bg-color-primary text-white hover:bg-color-primary-hover active:color-primary-active flex justify-center items-center border-transparent py-2 px-4 rounded-md font-semibold border"
>
  Hello
</button>
`;
